supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -137,19 +137,21 @@ If available, inform users of alternative capabilities and workarounds.
 [discrete]
 ===== Deprecation notice
 ----
-In <product> <release>, <name of capability> is deprecated. Bug fixes and support are provided through the end of the <releasename or, if unknown, “a future”> life cycle. After which, no new feature enhancements are made. (Optional: You can use <alternative capability> instead.)
+In <product> <release>, <name of capability or feature> is deprecated and is planned to be removed in the <deprecation time frame>. Bug fixes and support are provided through the end of the current release life cycle, but be aware that deprecated items do not receive further enhancements and will be removed. [Optional, if an alternative is available] Alternatively, you can use <alternative capability or feature> instead.
 ----
+When citing deprecation time frames, refer to specific releases (such as "the next release") only if that time frame is known to be accurate. Otherwise, refer to "a future release" because it gives flexibility to account for changes to the planned deprecation.
+
 .Example deprecation notice doc text
 ----
-In Red Hat Openstack Platform (RHOSP) 14, the director graphical user interface is deprecated. Bug fixes and support is provided through the end of a future life cycle but Red Hat do not implement new feature enhancements.
+In Red Hat Openstack Platform (RHOSP) 14, the director graphical user interface is deprecated and is planned to be removed in the next release. Bug fixes and support are provided through the current release life cycle, but be aware that deprecated items do not receive enhancements and will be removed.
```

Comment:
Avoid future release clause? See previous comment.

Reply:
Sentence:

> but be aware that deprecated items do not receive enhancements and will be removed.

Has no subject. Red Hat should be the subject here.

Reply:
> Sentence:
> 
> > but be aware that deprecated items do not receive enhancements and will be removed.
> 
> Has no subject. Red Hat should be the subject here.

The subject (you) is implied because it's an imperative.

Reply:
Hi @bburt-rh .

I disagree. Red Hat (team of engineers, and so on) is the correct subject to use here. The documentation user (usually an external customer) would not control enhancements to deprecated features. 

> but be aware that deprecated items do not receive enhancements and will be removed by Red Hat.

Reply:
@Amrita42 thanks, I'm not sure. (need to study some more about through vs throughout tbh!). And 'items' does seem a bit vague. I suppose it's to be broader than just products. Do you have a good alternative? 'components' maybe? maybe that's too specific though. If we don't have a better option I don't think 'items' causes any problems but I agree it's not great.

Reply:
Thanks @ndeevy I could not think of a replacement for **items**  maybe for now we can keep as is.
 I found this link >> https://wikidiff.com/through/throughout   >> imo we should use **throughout**  because we want to tel users that its during the entire duration of the release lifecycle.

Reply:
@bburt-rh wdyt about the rewrite?

Reply:
Once we have this comment addressed , we are good to go :)

Reply:
@Amrita42 @ndeevy @dfitzmau 
- I think that 'during' is better than `throughout` or `through`, so I will change it.
- I think `features` is better than `items`, and so I will change it.
- I will change the passive construction per the suggestion of @ndeevy 
- I'm rewriting to use future tense, because we're talking about the future.

Here's the update:

`Red Hat will provide bug fixes and support during the life cycle of the current release. Deprecated items will no longer receive enhancements and will be removed.`